day_7/10_little_monkeys.rb

```diff
@@ -0,0 +1,45 @@
+plural = ['Ten', 'Nine', 'Eight', 'Seven', 'Six', 'Five', 'Four', 'Three', 'Two']
+single = ['One']
+base = "jumping on the bed,
+One fell off and bumped his head,
+Mama called the doctor and the doctor said,
+\"No more monkeys jumping on the bed!\""
+
+plural.each do |words|
```

Comment:
@estein1988 - Nice work on iterating through the arrays here to print the appropriate rhyme! For line 8, I think it would make more sense to name the current item being iterated over (`|words|`) something singular such as `word`, `number`, etc.